src/Server.ts

```diff
@@ -1,22 +1,27 @@
 import * as net from "node:net";
 import EventEmitter from "node:events";
 import Packet from "./Packet.js";
+import path from "path";
+import { Config } from "./Config.js";
+import { PathLike } from "node:fs";
```

Comment:
```suggestion
import * as net from "node:net";
import EventEmitter from "node:events";
import { PathLike } from "node:fs";
import path from "node:path";
import Packet from "./Packet.js";
import { Config } from "./Config.js";
```

move node modules above our stuff just for better readability